examples/truffle/package.json

```diff
@@ -13,7 +13,7 @@
     "@truffle/hdwallet-provider": "^2.0.14"
   },
   "dependencies": {
-    "@oasisprotocol/sapphire-paratime": "workspace:^",
+    "@oasisprotocol/sapphire-paratime": "~1.1.1",
```

Comment:
we're pinning support for truffle to this version now? why not keep it 1.x, as we'll need to cut a 2.x at some point

Reply:
We plan on a patch release `1.1.2` with #175 and various other QOL issues.

This PR #191 will land in some `2.0.0` or `1.2.0` depending on how we like to name things.